### PR TITLE
Package fix: Victoria.Victoria

### PR DESCRIPTION
**Pull request by schembriaiden**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Changes Package Identifier and package name
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151932)

### CODE REVIEW

**Comment by vikingnope** [comment]
Don't exactly know what's wrong, should be fine

File: manifests/v/Victoria/Victoria/5.37/Victoria.Victoria.installer.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-Installers:
-- Architecture: x86
-  InstallerUrl: https://hdd.by/?download=1&kccpid=916&kcccount=https://hdd.by/Victoria/Victoria537.zip
-  InstallerSha256: A66B7869D59BBF90B5994332BAC57BCBC0B377CEE9F0C59C078795EA7B4C99D1
-ManifestType: installer
-ManifestVersion: 1.6.0
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Victoria.Victoria
```
Comment (stephengillie) [inline manifests/v/Victoria/Victoria/5.37/Victoria.Victoria.installer.yaml:4]: Automatic Validation ended with: 

>  Addition and deletion only permitted in the same package directory.

The pipelines identify manifest by `PackageIdentifier`, so this looks like a deletion of `Victoria.Victoria` and an addition of `Victora.HDD-SSD` in the same PR. This would disagree with the limit of 1 manifest (add/modify/remove) within the same PR. 

Could one of these changes be moved to another PR? Then both PRs should be able to pass this check.

Comment (vikingnope) [reply manifests/v/Victoria/Victoria/5.37/Victoria.Victoria.installer.yaml:4]: Ah I see, let me fix this then
